src/containers/SinglePost.js

```diff
@@ -22,7 +24,26 @@ SinglePost.propTypes = {
 
 export default compose(
   prefetch(({ dispatch, params }) => join(
-    dispatch(fetchPost(params.id)),
+    dispatch(fetchPost(params.id))
+    .then(action => {
+      let payload = action.payload
+      if (payload && !payload.api) {
+        let post = payload
+        let { media } = post
+        var metaTags = {
+          'og:title': post.name,
+          'og:description': truncate(striptags(post.description || ''), 140)
+        }
+        if (media[0]) {
+          metaTags = {...metaTags, ...{
```

Comment:
also i'm seeing a refactoring into a `setMetaTags (title, description, image)` function that can be used for both this and the project page. you could create a `util/index.js` and stick this in it, since i can't think of a better place under `util` to put it